util/topgen/lib.py

```diff
@@ -70,6 +70,25 @@ def get_module_by_name(top, name):
     return module
 
 
+def get_package_name_by_intermodule_signal(top, struct) -> str:
+    """Search inter-module signal package with the struct name
+
+    For instance, if `flash_ctrl` has inter-module signal package,
+    this function returns the package name
+    """
+    instances = top["module"] + top["memory"]
+
+    intermodule_instances = [
+        x["inter_signal_list"] for x in top["module"] + top["memory"]
```

Comment:
is `top["module"] + top["memory"]` supposed to be `instances`?

Reply:
right... I suppose to remove `instances` line...